src/snowflake/connector/config_manager.py

```diff
@@ -333,7 +334,14 @@ def read_config(
                     and filep.stat().st_uid != os.getuid()
                 )
             ):
-                warn(f"Bad owner or permissions on {str(filep)}")
+                # for non-Windows, suggest change to 0600 permissions.
+                chmod_message = (
+                    f". To change owner, run chown $USER {str(filep)}. To restrict permissions, run chmod 0600 {str(filep)}."
```

Comment:
Surround the commands with some sort of quotation marks, please
```suggestion
                    f". To change owner, run `chown $USER {str(filep)}`. To restrict permissions, run `chmod 0600 {str(filep)}`."
```